bin/ghe-backup

```diff
@@ -94,7 +94,7 @@ if ghe-ssh "$GHE_HOSTNAME" -- \
 fi
 
 # Log backup start message in /var/log/syslog on remote instance
-ghe_remote_logger "Starting backup from $(hostname) in snapshot $GHE_SNAPSHOT_TIMESTAMP ..."
+ghe_remote_logger "Starting backup from $(hostname) / snapshot $GHE_SNAPSHOT_TIMESTAMP ..."
```

Comment:
I think it's clearer with "in". With the slash it looks like backing up _from_ hostname/snapshot.